openhtf/exe/phase_executor.py

```diff
@@ -55,6 +57,16 @@
 class InvalidPhaseResultError(Exception):
   """Raised when a PhaseOutcome is created with an invalid phase result."""
 
+class ExceptionOutcome(collections.namedtuple('ExceptionOutcome', ['exc_type','exc_val','exc_tb'])):
+
+  def _asdict(self):
+    return_dict = {}
+    excep = traceback.format_exception(self.exc_type,self.exc_val,self.exc_tb)
+    return_dict['exc_type'] = str(self.exc_type)
+    return_dict['exc_val'] = self.exc_val
+    return_dict['exc_tb'] = "".join(a for a in excep)
```

Comment:
single instead of double quotes.  also, did you need to do `a for a in excep`, or could you just do `excep`?  I think `join()` takes an iterable directly.